cfpb/cfgov-refresh
# Standing up the YES transportation tool

This PR adds the basic building blocks of the new YES transportation tool. There are a lot of upfront changes and I'd like to get eyes / approval on it before I add much more!

## Additions

- Entry point for app in `base.py` settings
- Folders in `unprocessed/apps` for static content and in `cfgov` for templates and python files
- Minimal python files, including a migration, for a new page type. This code could probably be removed in a new PR. I'm not sure if we want to introduce a migration before we know all the content the new page type might be storing.
- Templates for the `budget` and `transportation tool` sections. The templates collectively represent a single page.
- some CSS extensions
- JS files to cover a few views, basic state management, and tests


## Removals

-

## Changes

-

## Testing

Apologies if any of these steps are off base, I'm not super familiar with the exact order of things in wagtail!

1. Run migration included in this PR
2. Add a new child page to  `/practitioner-resources/resources-youth-employment-programs` called 'transportation-tool', with a page type of `TransportationTool`
3. Build assets and start the app
4. You _should_ be able to see the current page and interact with the form fields (most only store internal state at this point)

## Screenshots


## Notes

- We don't want the page to be live at this point. This PR is for adding visibility to the code, and to get something in the repo before too much more content is added.

- We don't have 'final' designs so usability without JS or CSS isn't possible yet; this is more about standing something up

## Todos

- Modify`BudgetFormView` to use `InputView` component in a future PR

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
